sdk/automation/Azure.ResourceManager.Automation/src/Generated/Models/LinuxUpdateClass.cs

```diff
@@ -10,7 +10,10 @@
 
 namespace Azure.ResourceManager.Automation.Models
 {
-    /// <summary> Update classifications included in the software update configuration. </summary>
+    /// <summary>
+    /// Update classifications included in the software update configuration.
+    /// Serialized Name: LinuxUpdateClasses
+    /// </summary>
     public readonly partial struct LinuxUpdateClass : IEquatable<LinuxUpdateClass>
```

Comment:
Suggest renaming to `LinuxUpdateClassification`

Reply:
fixed